### PR TITLE
fix: Remove unused fixtures and fix test name in smart swaps disabled spec

### PR DESCRIPTION
**Pull request by plasmacorral**
## **Description**

Test name was not being captured correctly and recently came to appreciate that some of the snap account related fixtures were not actually needed for this test.  

Updated test name and removed `accountSnapFixtures`

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25616?quickstart=1)

## **Related issues**

Fixes:  [505](https://github.com/MetaMask/accounts-planning/issues/505)

## **Manual testing steps**
Firefox: 
`yarn build:test:mv2` then `ENABLE_MV3=false yarn test:e2e:single test/e2e/accounts/smart-swap-disabled.spec.ts --browser=firefox`
Chrome:
`yarn build:test` then `yarn test:e2e:single test/e2e/accounts/smart-swap-disabled.spec.ts --browser=chrome`

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: test/e2e/accounts/smart-swap-disabled.spec.ts [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 import { Driver } from '../webdriver/driver';
-import {
-  accountSnapFixtures,
-  installSnapSimpleKeyring,
-  makeNewAccountAndSwitch,
-} from './common';
+import { installSnapSimpleKeyring, makeNewAccountAndSwitch } from './common';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+const FixtureBuilder = require('../fixture-builder');
```
Comment (gantunesr) [inline test/e2e/accounts/smart-swap-disabled.spec.ts:7]: Any reason on why you can't just import using the required syntax?

```suggestion
import FixtureBuilder from '../fixture-builder';
```

**Review by gantunesr — COMMENTED** [review]
Since you're already changing this file, lets update the name of the test

```ts
describe('Snap Account - Smart Swaps', function (this: Suite) {
  it('checks if smart swaps are disabled for snap accounts', async function () {
    ...
  }
}

```

File: test/e2e/accounts/smart-swap-disabled.spec.ts (hunk 1 above)
Comment (plasmacorral) [reply test/e2e/accounts/smart-swap-disabled.spec.ts:7]: No? Good question/suggestion.  Updated.  Thanks

**Review by chloeYue — APPROVED** [review]
LGTM! Thanks for adjusting this !

**Review by seaona — APPROVED** [review]

**Pull request merged**